aten/src/ATen/OpaqueTensorImpl.h

```diff
@@ -70,7 +70,11 @@ struct TORCH_API OpaqueTensorImpl : public TensorImpl {
       const c10::VariableVersion& version_counter,
       bool allow_tensor_metadata_change) const override {
     auto impl = c10::make_intrusive<OpaqueTensorImpl<OpaqueHandle>>(
-        key_set(), dtype(), device(), opaque_handle_, sizes_and_strides_.sizes_arrayref());
+        key_set(),
+        dtype(),
+        device(),
+        opaque_handle_,
+        expectIntArrayRef(sizes_and_strides_.sizes_arrayref()));
```

Comment:
If there is an invariant that OpaqueTensorImpl never has symbolic ints this should be an unchecked cast, but this path is not perf critical so it seems ok